src/web/components/SharingPermission/ParticipantSearchBar.tsx

```diff
@@ -2,28 +2,31 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import clsx from 'clsx';
 import { ChangeEvent, useState } from 'react';
 
+import { ParticipantTypeDTO } from '../../../api/entities/ParticipantType';
 import { AvailableParticipantDTO } from '../../../api/participantsRouter';
-import { ParticipantTypeResponse } from '../../services/participantType';
 import { ParticipantsTable } from './ParticipantsTable';
 import { TypeFilter } from './TypeFilter';
 
 import './ParticipantSearchBar.scss';
 
 type ParticipantSearchBarProps = {
   participants: AvailableParticipantDTO[];
-  participantTypes: ParticipantTypeResponse[];
+  participantTypes: ParticipantTypeDTO[];
   selectedParticipantIds?: Set<number>;
   onSelectedChange: (selectedItems: Set<number>) => void;
+  open: boolean;
+  onToggle: (open: boolean) => void;
```

Comment:
I wonder if we can rename this prop to be more clear - what is is toggling? maybe `onToggleOpen`?

Reply:
agree, changed